### PR TITLE
Dark theme for changelog and completed upgrades

### PR DESCRIPTION
**Pull request by dengr1065**
really quick fix just 3 sloc

### CODE REVIEW

File: src/css/ingame_hud/shop.scss [hunk 1]
```diff
@@ -241,6 +241,10 @@
 
                             &.complete {
                                 background-color: $colorGreenBright;
+                                
+                                @include DarkThemeOverride {
+                                    background-color: $colorGreenBright;
+                                }
```
Comment (tobspr) [inline src/css/ingame_hud/shop.scss:247]: What is this doing here?

Comment (dengr1065) [reply src/css/ingame_hud/shop.scss:247]: DarkThemeOverride for parent (`progressBar` or something like that) for some reason takes over `.complete` background color, even though it should not; so applying color here as well fixes it.

**Comment by dengr1065** [comment]
marked as resolved because no feedback

**Comment by tobspr** [comment]
Thank you!

**Pull request merged**